### PR TITLE
Removed reference to PollableChannel

### PR DESCRIPTION
**Pull request by nitishasgekar**
Removed reference to PollableChannel as per https://github.com/spring-cloud/spring-cloud-stream/issues/436#issuecomment-205430607

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@nitishasgekar Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-stream&pullRequestId=1003)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-cloud/spring-cloud-stream&pullRequestId=1003) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@nitishasgekar Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-stream&pullRequestId=1003)!

File: spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc [hunk 1]
```diff
@@ -212,7 +212,7 @@ The `@EnableBinding` annotation can take as parameters one or more interface cla
 
 [NOTE]
 ====
-In Spring Cloud Stream 1.0, the only supported bindable components are the Spring Messaging `MessageChannel` and its extensions `SubscribableChannel` and `PollableChannel`.
+In Spring Cloud Stream 1.0, the only supported bindable components are the Spring Messaging `MessageChannel` and its extension `SubscribableChannel`.
```
Comment (mbogoevici) [inline spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc]: We should also remove the 1.0 version.

**Comment by ilayaperumalg** [comment]
@nitishasgekar Thank you for the contribution!

LGTM, merging.

File: spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc [hunk 2]
```diff
@@ -212,7 +212,7 @@ The `@EnableBinding` annotation can take as parameters one or more interface cla
...
 Future versions should extend this support to other types of components, using the same mechanism.
```
Comment (ilayaperumalg) [inline spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc]: Given we have `KStream` components ready to be merged, we can refer it here. We can track it via https://github.com/spring-cloud/spring-cloud-stream/issues/1019

File: spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc (hunk 1 above)
Comment (ilayaperumalg) [inline spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc]: For the context of this PR, I will update this line as:

In Spring Cloud Stream, the only supported **bindable MessageChannel** components are the Spring Messaging `MessageChannel` and its extension `SubscribableChannel`.`

**Comment by ilayaperumalg** [comment]
Rebased and merged as de286f5e6a01161da5d70a38cbcbff188c60220a

**Pull request closed without merging**